Open Judge System/Services/OJS.Services/RedisCacheService.cs

```diff
@@ -188,5 +218,22 @@ public async Task RemoveAsync(string cacheId)
                    expiration);
             }
         }
+
+        private bool MemoryCacheHasKey(string key, string value)
+        {
+            if (HttpRuntime.Cache.Get(key) == null)
+            {
+                HttpContext.Current.Cache.Add(
+                   key,
+                   value,
+                   null,
+                   DateTime.Now.AddMinutes(this.memmoryCacheExpirationInMinutes),
+                   TimeSpan.Zero,
```

Comment:
Ins't the `ICacheService` that you deleted useful for this case? You can bring it back and rename it to `IMemoryCacheService`

Reply:
In the ICacheService  there is a number of methods which are not used and will be removed anyway. There will be only one method which for now is called in only one place inside the RedisCacheService. Should we return an interface and implementation just for this method